test/mox_test.exs

```diff
@@ -351,39 +351,84 @@ defmodule MoxTest do
 
       verify!()
       expect(CalcMock, :add, fn x, y -> x + y end)
+      expect(SciCalcOnlyMock, :exponent, fn x, y -> x * y end)
 
-      message = ~r"expected CalcMock.add/2 to be invoked once but it was invoked 0 times"
-      assert_raise Mox.VerificationError, message, &verify!/0
+      assert_raise(Mox.VerificationError, &verify!/0)
+      try do
+        verify!()
+      rescue
+        error in Mox.VerificationError ->
+          assert error.message =~ ~r"expected CalcMock.add/2 to be invoked once but it was invoked 0 times"
+          assert error.message =~ ~r"expected SciCalcOnlyMock.exponent/2 to be invoked once but it was invoked 0 times"
+      end
 
-      assert CalcMock.add(2, 3) == 5
+      CalcMock.add(2, 3)
```

Comment:
**Annotation:** I remove assertions of the `CalcMock.add` method because, again, these tests should be focusing on assertions related to the `verify` method **not** the return behavior introduced by the `expect` method call.